.fixtures.yml

```diff
@@ -20,6 +20,8 @@ fixtures:
     pe_r10k: "git@github.com:puppetlabs/puppetlabs-pe_r10k.git"
     pe_staging: "git@github.com:puppetlabs/puppetlabs-pe_staging.git"
     puppet_enterprise: "git@github.com:puppetlabs/puppetlabs-puppet_enterprise.git"
+    stunnel: "git@github.com:puppetlabs/puppetlabs-stunnel.git"
```

Comment:
Ha, I guess I used git urls for the others. I'll make a ticket to review this and decide on the best strategy.